spec/harvesting/client_spec.rb

```diff
@@ -539,6 +539,62 @@
         end
       end
     end
+  end
+
+  describe "#user_assignments", :vcr do
+    context "as an admin user" do
+      subject { Harvesting::Client.new(access_token: admin_access_token, account_id: account_id) }
+
+      it 'retreives the accounts user assignments' do
+        user_assignments = subject.user_assignments
+
+        projects = user_assignments.map { |ua| ua.project.id }.uniq
+        expect(projects).to contain_exactly(
+          project_castle_building.id,
+          project_road_building.id
+        )
+
+        users = user_assignments.map { |ua| ua.user.id }.uniq
+        expect(users).to contain_exactly(
+          user_john_smith.id,
+          user_jane_doe.id,
+          user_me.id
+        )
+      end
+
+      it 'retrieves project user assignments for castle building project' do
+        user_assignments = project_castle_building.user_assignments
+        users = user_assignments.map { |ua| ua.user.id }.uniq
+        expect(users).to contain_exactly(user_john_smith.id, user_me.id)
+      end
```

Comment:
@mscottford Could you move this spec to the `project_spec.rb` file? I think it makes more sense over there...

Reply:
Done!